store/README.md

```diff
@@ -0,0 +1,167 @@
+# Store
+
+**Store** maintains the state of the chain. It serves as the "source of truth" for the chain - i.e., if it is not in 
+the store, the node does not consider it to be a part of the chain. 
+**Store** is one of components of the [Miden node](..).
+
+## Architecture
+
+The Miden node is still under heavy development and current architecture is subject of change. This topic will be 
+filled later.
+
+## Usage
+
+### Installing the Store
+
+Store can be installed and run as a part of [Miden node](../README.md#installing-the-node).
+But if you intend on running Store as a separated process, you need to install and run it separately:
+
+```sh
+# Installs `miden-node-store` executable
+cargo install --path store
+```
+
+In order to run Store, you must provide genesis file. To generate genesis file you need to use [Miden node](../README.md#generating-the-genesis-file)'s `make-genesis` command. 
```

Comment:
Change to: 

In order to run the Store, you must provide a genesis file. To generate a genesis file you will need to use [Miden node](../README.md#generating-the-genesis-file)'s `make-genesis` command. 


You will also need to provide a configuration file. We have an example config file in [store-example.toml](store-example.toml).